config/environment.js

```diff
@@ -23,7 +23,7 @@ module.exports = function(environment) {
       'script-src': '\'self\' www.google-analytics.com www.googletagmanager.com',
       'font-src': '\'self\' fonts.gstatic.com',
       'connect-src': '\'self\' sentry.io',
-      'img-src': '\'self\' camo.csvalpha.nl www.google-analytics.com img.youtube.com',
+      'img-src': '\'self\' camo.csvalpha.nl www.google-analytics.com img.youtube.com data:',
```

Comment:
I am not sure if I like this, this is potentially insecure.

```
data: Allows data: URIs to be used as a content source. This is insecure; an attacker can also inject arbitrary data: URIs. Use this sparingly and definitely not for scripts.
```
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src

Reply:
Hmm that's true... Is there any other way to preview the uploaded images, without adding :data to the cors policy?

Reply:
Maybe blobs? Or check with dropzone itself what the recommend

Reply:
Well, it is a blob, which is set as the image source as a base64 encoded string. Don't know if there's any other way to set the image source from a blob, but then you basically have the same problem again as described on that site, don't you?

Reply:
I have created a issue with the package, wondering how other developers handle with this. https://github.com/FutoRicky/ember-cli-dropzonejs/issues/112

Reply:
Great, we'll see what they'll respond with. For the time being, I removed :data from the CSP. For now we'll then at least have a working dropzone, just without preview images.